internal/core/model/adapters.rs

```diff
@@ -16,7 +16,7 @@ use super::*;
 /// ## Example
 ///
 /// Here we have a [`VecModel`] holding rows of a custom type `Name`.
-/// It is then mapped into a `MapModel` of [`SharedString`]s
+/// It is then mapped into a `MapModel` of [`crate::SharedString`]s
```

Comment:
Hm, do you remember why this is needed?

Reply:
Ah, I see, you removed the use of `SharedString` and replaced it with qualified use. Do you remember why?

Reply:
I think the documentation looks better without the `crate::`

Reply:
Yes without that I get an unused warning. Before `SharedString` was also used on an other place and therefore before there was no warning.

Reply:
@ogoffart is there a way to handle that without the unused warning? with `allow_unused` flag?

Reply:
Probably there is a way, yes.
How about continuing to use `SharedString` directly, also in the `impl From<SharedString>`  ?